hack/ci/setup-k8s.sh

```diff
@@ -5,7 +5,7 @@ set -eux
 # This image tag corresponds to a Kubernetes version that kind installs using
 # images at:
 # https://hub.docker.com/r/kindest/node/tags
-K8S_VERSION="v1.15.4"
+K8S_VERSION=$1
 KIND_IMAGE="quay.io/estroz/node:${K8S_VERSION}"
 # TODO: use the below image once it is rebuilt with the following base image:
 # kindest/base:v20190926-e6b6f7f0@sha256:1ec92b2910f2bfb8a4814cc90e15974a499f0c93d203b879277fa4bdb3762ce2
```

Comment:
Also, note that at the end it needs to be a matrix in the trevis which the same tests will be execute for each version such as:

<img width="1271" alt="Screenshot 2019-11-14 at 01 52 37" src="https://user-images.githubusercontent.com/7708031/68819952-9c3cce80-0681-11ea-87bb-66b6479b8c66.png">